samples/Azure.Analytics.Purview.Account/Generated/PurviewAccountCollections.cs

```diff
@@ -59,41 +59,25 @@ internal PurviewAccountCollections(ClientDiagnostics clientDiagnostics, HttpPipe
         /// <summary> Get a collection. </summary>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <remarks>
+        /// Below is the JSON schema for the request and response payloads.
         /// Schema for <c>Response Body</c>:
```

Comment:
This is showing up in the rendered documentation as being on the same line as the previous comment.  Let's make sure there is a carriage return separating these in the rendered documentation.

![image](https://user-images.githubusercontent.com/21269347/174655592-af8ae824-5d80-438e-a019-5575476026b1.png)